tests/resources/technical-records.json

```diff
@@ -17724,7 +17724,10 @@
           "code": "v",
           "description": "3 wheelers"
         },
-        "vehicleConfiguration": "other",
+        "vehicleConfiguration": null,
```

Comment:
Why was this change needed?

Reply:
I think I changed it as an experiment, changed back.